code/__DEFINES/magic.dm

```diff
@@ -29,3 +28,12 @@
 #define INVOCATION_EMOTE "emote"
 ///Forces the wizard to whisper (and be able to) to cast the spell.
 #define INVOCATION_WHISPER "whisper"
+
+/// Default magic resistance that blocks normal magic (wizard, spells, magical staff projectiles)
+#define MAGIC_RESISTANCE (1<<0)
+/// Tinfoil hat magic resistance that blocks mental magic (telepathy, mind curses, abductors, jelly people)
+#define MAGIC_RESISTANCE_MIND (1<<1)
+/// Holy magic resistance that blocks unholy magic (revenant, cult, vampire, voice of god)
+#define MAGIC_RESISTANCE_HOLY (1<<2)
+/// All magic resistances combined
+#define MAGIC_RESISTANCE_ALL (MAGIC_RESISTANCE|MAGIC_RESISTANCE_MIND|MAGIC_RESISTANCE_HOLY)
```

Comment:
Can this just be an `ALL` bitflag (`~0`) or will that break things? Just a thought

Reply:
I refactored `MAGIC_RESISTANCE_ALL` to just use `ALL` where it was used in the code and removed the define with https://github.com/tgstation/tgstation/pull/64124/commits/3df7f8092332af0cf6f0e043166e3a021ed34a86